frontend/packages/console-app/src/actions/creators/pdb-factory.ts

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react';
+import i18next from 'i18next';
+import * as _ from 'lodash';
+import { Action } from '@console/dynamic-plugin-sdk';
+import { asAccessReview } from '@console/internal/components/utils';
+import { useK8sWatchResource } from '@console/internal/components/utils/k8s-watch-hook';
+import { PodDisruptionBudgetModel } from '@console/internal/models';
+import {
+  K8sResourceCommon,
+  K8sPodControllerKind,
+  K8sKind,
+  PodDisruptionBudgetKind,
+  referenceForModel,
+} from '@console/internal/module/k8s';
+import { useActiveNamespace } from '@console/shared';
+import { deletePDBModal } from '../../components/pdb/modals';
+import { getPDBResource } from '../../components/pdb/utils/get-pdb-resources';
+import { ResourceActionFactory } from './common-factory';
+
+const pdbRoute = ({ metadata: { name, namespace } }: K8sResourceCommon, kindObj: K8sKind) =>
+  `/k8s/ns/${namespace}/${referenceForModel(kindObj)}/form?name=${name}&kind=${kindObj.kind}`;
```

Comment:
@andrewballantyne I don't think so. The `name` and `kind` are URL query parameters used for fetching pod controller.

Reply:
🤔  Not sure if that's the case Cyril... you use it in the `useK8sWatchResource` which needs to be a qualified path to the kind so as not to conflict with kinds of the same name but a different group/version.

https://github.com/openshift/console/pull/10445/files#diff-7c0f976f8ff3c37df4d6ee988c85e5fb2740529beeb98a2eb4bd4393fdd554d7R37

Reply:
> 

Yes, the query parameters are used in the `useK8sWatchResource` to fetch target pod controller. However I see what you are referring to now - `groupVersionKind`. This would include adding `group` and `version`  in the url.